app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  has_many :posts, class_name: 'Post', foreign_key: :author_id
+  has_many :likes, foreign_key: :users_id
+  has_many :comments
```

Comment:
- [ ] For user model, it should also have `comments` with `foreign_key: 'author_id'`

``has_many :comments, foreign_key: 'author_id'``